packages/components/src/components/hds/layout/grid/index.ts

```diff
@@ -104,6 +113,15 @@ export default class HdsLayoutGrid extends Component<HdsLayoutGridSignature> {
     const inlineStyles: {
       '--hds-layout-grid-column-min-width'?: string;
       '--hds-layout-grid-column-fill-type'?: string;
+
+      // responsize
+      '--hds-layout-grid-column-width-sm'?: string;
+      '--hds-layout-grid-column-width-md'?: string;
+      '--hds-layout-grid-column-width-lg'?: string;
+
+      '--hds-layout-grid-column-fill-type-sm'?: string;
```

Comment:
[question] since there is already a "default" `auto-fill` value for `--hds-layout-grid-column-fill-type` in CSS, why do we need these extra values, that are assigned `auto-fill` anyway in JavaScript below?